### ISSUE TITLE
What's the length setting of Table 1?

### ISSUE THREAD

**Issue by Spico197** [swj0419/detect-pretrain-code#7]
Hi there, thanks for publishing such an amazing work!

I'm trying to reproduce the results reported in the paper, and I'm wondering how are the results in Table 1 calculated.
Are they just the macro-averaged scores of different lengths, or the experiments are conducted under the default length=64?

Thanks very much for your reply~

![image](https://github.com/swj0419/detect-pretrain-code/assets/22840952/a14d470e-0578-4230-9dc8-288e02c5f17f)

**Comment by zjysteven** [comment]
Hey @Spico197 have you figured this out by any chance?

**Comment by Spico197** [comment]
> Hey @Spico197 have you figured this out by any chance?

Hi there, sorry for the late response. I've not figured out this yet. I think a trial on Pythia-2.8B may help us resolve this?

**Comment by kirklandWater1** [comment]
@Spico197 @zjysteven Did anyone find out the length yet? @swj0419

**Comment by zjysteven** [comment]
Yeah it should be 128 based on my reproduction results (code is available [here](https://github.com/zjysteven/mink-plus-plus)).
